docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '3.9'
 services:
   api:
     ports:
-      - "3000:3000"
+      - "3000:3001"
```

Comment:
```suggestion
```

Reply:
We should instead use something else like Traefik. For now, ok, we will drop that change.